test/sanityQueryService.test.js

```diff
@@ -293,4 +293,22 @@ describe('Sanity Queries', function () {
         const response = await fetchScheduledReleases('drumeo', {});
         expect(response.length).toBeGreaterThan(0);
     });
+
+    test('fetchAll-GroupBy-Genre', async () => {
+        let response = await fetchAll('drumeo', 'solo',{groupBy: 'genre'});
+        console.log(response);
+        expect(response.entity[0].web_url_path).toContain('/drumeo/genres/');
+    });
```

Comment:
More of a question to @adrian-musora since he has more experience with the tests here:
1. Is there a format that the test names should follow? e.g. in the Laravel tests, we try to do something `test_order_updated_dispatches_shopify_sync_customer_job_for_post_launch` so that when reading the test output it's clear what the test is doing: "order updated dispatches shopify sync customer job for post launch".
Seeing the other tests above these, I see others like "fetchAll-IncludedFields" so I'll assume these names are fine.
2. Would there be value in checking each entry in the response.entity's web_url_path, or is it sufficient to just check the first entry?

Reply:
No specific naming convention. I've been using `methodName-paramaterization` as the naming scheme. So this would be a valid name. 
Most of the tests only verify execution and don't really verify the results package contains valid data.